src/verify.c

```diff
@@ -267,6 +270,7 @@ static int get_required_files(struct manifest *official_manifest, struct list *s
 	if (ret) {
 		error("Unable to download necessary files for this OS release\n");
 	}
+	progress_complete_step();
```

Comment:
this step should have been completed automatically too after downloading the files, I don't think you need this one.

Reply:
No we dont. But I'll move this in another commit